server/endpoints/models/PT.js

```diff
@@ -0,0 +1,59 @@
+const { Double } = require('mongodb');
+const mongoose = require('mongoose');
+const Schema = mongoose.Schema;
+
+const PTSchema = new Schema({
+  username: {
+    type: String,
+    required: true
+  },
+  email: {
+    type: String,
+    required: true
+  },
+  publicName: {
+    type: String,
+    required: true
+  },
+  language: {
+    type: String,
+    required: true
+  },
+  postiveRatingCount: {
+    type: Integer
+  },
+  negativeRatingCount: {
+    type: Integer
+  },
+  location: {
+    type: String
+  },
```

Comment:
Can we have location be specified by city and state? That will probably make it easier for clients to query PTs based on location.
I.e. const Location = new Schema({ city: String, state: String })

Side thought: Do we want clients to be able to find PTs near them, based on distance? I'm not sure if distance matters bc everything is online, but if so, then we should probably also add geospatial data (GeoJSON) to location. 
I.e. PT location is a Point with coordinates.
LocationCoord: { type: "Point", coordinates: [x, y] }
And then we can find PTs within a certain distance from the client's location with $near.

-- (Probably obvious, but these considerations also apply to the Client schema)

Reply:
Changed to a GeoJSON Point.